src/JWT/Algorithms/RS256Algorithm.cs

```diff
@@ -19,8 +19,8 @@ public sealed class RS256Algorithm : IJwtAlgorithm
         /// <param name="privateKey">The RSA key for signing the data.</param>
         public RS256Algorithm(RSACryptoServiceProvider publicKey, RSA privateKey)
         {
-            _publicKey = publicKey ?? throw new InvalidOperationException("Private key is null");
-            _privateKey = privateKey ?? throw new InvalidOperationException("Public key is null");
+            _publicKey = publicKey ?? throw new InvalidOperationException("Public key is null");

```

Comment:
Can you please change `InvalidOperationException` to `ArgumentNullException(nameof(paramName)`

Reply:
done!